tutorials/getting_started/model.md

```diff
@@ -207,30 +207,37 @@ Where that's not supported, you can make the changes through the Item's Page.
-| Climate      | Status      | HVAC                                                       | Status, Control, Switch            | _Any_       |
-| Screens      | Status      | Screen, Television                                         | Status, Control, Switch            | Power       |
-| Speakers     | Status      | Receiver, Speaker                                          | Status, Control, Switch            | Power       |
-| Projectors   | Status      | Projector                                                  | Status, Control, Switch            | Power       |
-| Alarms       | Status      | _Any_                                                      | Alarm                              | _Any_       |
-| Luminance    | Measurement | _Any_                                                      | Measurement                        | Light       |
-| Temperature  | Measurement | _Any_                                                      | Measurement                        | Temperature |
-| Humidity     | Measurement | _Any_                                                      | Measurement                        | Humidity    |
+| Badge        | Type        | Equipment                                                  | Point                              | Property           |
+|--------------|-------------|------------------------------------------------------------|------------------------------------|--------------------|
+| Low Battery  | Status      | _Any_                                                      | _Any_                              | LowBattery         |
+| Lights       | Status      | LightSource, children                                      | _Any_                              | Light              |
+| Windows      | Status      | Window                                                     | _Any_                              | Opening, children  |
+| Doors        | Status      | Door, FrontDoor, BackDoor, InnerDoor, CellarDoor, SideDoor | _Any_                              | Opening, children  |
+| Garage Doors | Status      | GarageDoor                                                 | _Any_                              | Opening, children  |
+| Blinds       | Status      | WindowCovering, children                                   | _Any_                              | Opening, children  |
+| Presence     | Status      | _Any_                                                      | _Any_                              | Presence, Motion   |
+| Lock         | Status      | Lock                                                       | _Any_                              | Opening, children  |
+| Climate      | Status      | HVAC, children                                             | Status, Control, Switch            | _Any_              |
+| Screens      | Status      | Screen, Television                                         | Status, Control, Switch            | Power              |
```

Comment:
Is this still the right tag for it, or should we rather use Display and all its children? I didn't check the recommendations for it.

Reply:
I don't know. For some reason the original code differentiates between displays, screens, tvs on one side and projectors on the other. Personally I would lump them all together under one badge.

Reply:
That’s also my view.

Reply:
But I vote to leave the current collections unchanged.

Reply:
Maybe use all from Display + Screen.